src/assets/selection_polygon.svg

```diff
@@ -0,0 +1,50 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Did you draw this yourself, or should you include a thankyou?

Reply:
I drew it in inkscape this morning.